lib/rules/array-bracket-spacing.js

```diff
@@ -160,21 +115,37 @@ module.exports = function(context) {
 
 module.exports.schema = [
     {
-        "enum": ["always", "never"]
+        "enum": ["always", "never"],
+        "default": "never"
     },
     {
         "type": "object",
         "properties": {
             "singleValue": {
-                "type": "boolean"
+                "type": "boolean",
+                "default": false
             },
             "objectsInArrays": {
-                "type": "boolean"
+                "type": "boolean",
+                "default": false
             },
             "arraysInArrays": {
-                "type": "boolean"
+                "type": "boolean",
+                "default": false
             }
         },
         "additionalProperties": false
     }
 ];
+
+module.exports.meta = {
+    errors: {
+        extraSpaceAfter: "There should be no space after '{{token}}'",
```

Comment:
Ok, that was just one of the approaches. I'll try to think of some other way, and submit another pull request.